packages/core/lib/command-utils.js

```diff
@@ -116,11 +119,11 @@ const getCommand = ({ inputStrings, options, noAliases }) => {
   // for inferring the command.
   if (firstInputString === "-v" || firstInputString === "--version") {
     chosenCommand = "version";
-  } else if (commands.includes(firstInputString)) {
+  } else if (validTruffleCommands.includes(firstInputString)) {
```

Comment:
Isn't this function called in both shell and REPL contexts?

Reply:
I missed that, yuck! The REPL's use is [guarded by `validTruffleConsoleCommands`](https://github.com/trufflesuite/truffle/blob/help-less/packages/core/lib/console.js#L401-L406).  It seems redundant to invoke `getCommand()` if it is already a valid console command. I'm going to remove that.